public/app/features/explore/LogsContainer.tsx

```diff
@@ -195,16 +167,16 @@ function mapStateToProps(state: StoreState, { exploreId }: { exploreId: string }
     absoluteRange,
     dedupStrategy,
   } = item;
-  const dedupedRows = deduplicatedRowsSelector(item);
+  const dedupedRows = deduplicatedRowsSelector(item) || undefined;
   const timeZone = getTimeZone(state.user);
 
   return {
     loading,
     logsHighlighterExpressions,
-    logRows: logsResult && logsResult.rows,
```

Comment:
If logsResult was `null,` then logRows (and everything under) was `null` (as `null && null.rows` results in `null`).  However, in props we were expecting values or undefined. This became a source of errors in live tailing (see below). So I updated it to return value or undefined.